buzsaki_lab_to_nwb/original_code/test_add_unit.py

```diff
@@ -1,13 +0,0 @@
-from pynwb import NWBFile, NWBHDF5IO
-from datetime import datetime
-
-
-nwbfile = NWBFile('a', 'b', 'c', datetime.now(), datetime.now())
-
-nwbfile.add_unit_column('excitatory_connections', 'ids of cells that receive excitatory input from cell')
-
-nwbfile.add_unit({'id': 1, 'excitatory_connections': [1, 2]})
-nwbfile.add_unit({'id': 2, 'excitatory_connections': [1, 2, 3]})
-
-with NWBHDF5IO('test_unit.nwb', 'w') as io:
-    io.write(nwbfile)
```

Comment:
I removed several of these little example test scripts (and also the incomplete matlab converter version) that didn't appear as if they did anything major for the actual Yuta conversion. Please let me know if you want any of them added back in.